.github/workflows/bump-version.yml

```diff
@@ -0,0 +1,52 @@
+name: Bump version
+
+on:
+  workflow_call:
+    inputs:
+      tag-prefix:
+        required: false
+        default: 'v'
+        type: string
+    outputs:
+      tag:
+        description: Newly created tag
+        value: ${{ jobs.bump-version.outputs.tag }}
+
+jobs:
+  bump-version:
+    name: Bump Version on master
```

Comment:
This action will have permission to merge to master.? 🤔

Reply:
It works on regular branch. But just tested, and it does not work on protected branch, it requires a `Personal Access Token`, instead of `GITHUB_TOKEN`